shared/versions/stack/7.x.asciidoc

```diff
@@ -1,14 +1,14 @@
-:version:                7.9.0
+:version:                7.10.0
```

Comment:
This got me thinking, can we just make this whole file `include::7.10.asciidoc[]`?

Reply:
Ignore this, `branch` is different